### PR TITLE
Enable AppVeyor builds for Windows (unit tests)

### PR DESCRIPTION
**Pull request by sstone1**
Contributes to #170

Signed-off-by: Simon Stone <sstone1@uk.ibm.com>

### CODE REVIEW

File: client/test/commands/createSmartContractProjectCommand.test.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ describe('CreateSmartContractProjectCommand', () => {
     let executeCommandStub: sinon.SinonStub;
     let uri: vscode.Uri;
     let uriArr: Array<vscode.Uri>;
-    const USER_TEST_DATA = path.join(path.dirname(__dirname), '../../test/data');
+    const USER_TEST_DATA = path.join(path.dirname(__dirname), '..', '..', 'test', 'data');
```
Comment (caroline-church) [inline client/test/commands/createSmartContractProjectCommand.test.ts]: type?

File: client/package.json [hunk 2]
```diff
@@ -297,7 +297,7 @@
         "postinstall": "node ./node_modules/vscode/bin/install",
         "pretest": "npm run compile && npm run lint && npm run licchk && npm run updatePackageJSON",
         "test": "node ./node_modules/vscode/bin/test --verbose",
-        "posttest": "node ./node_modules/.bin/remap-istanbul -i ./coverage/coverage.json -o ./coverage/coverage-remap.json && node ./node_modules/.bin/istanbul check-coverage ./coverage/coverage-remap.json",
+        "posttest": "remap-istanbul -i ./coverage/coverage.json -o ./coverage/coverage-remap.json && istanbul check-coverage ./coverage/coverage-remap.json",
```
Comment (caroline-church) [inline client/package.json]: does it still find the command?

File: client/test/commands/createSmartContractProjectCommand.test.ts [hunk 3]
```diff
@@ -384,7 +384,7 @@ describe('CreateSmartContractProjectCommand', () => {
         sendCommandStub.onCall(0).resolves();
 
         sendCommandStub.onCall(1).resolves('0.0.7');
-        const wrongPath = path.join(path.dirname(__dirname), '../../test/data/nonexistent');
+        const wrongPath = path.join(path.dirname(__dirname), '..', '..', 'test', 'data', 'nonexistent');
```
Comment (caroline-church) [inline client/test/commands/createSmartContractProjectCommand.test.ts]: type?

File: client/package.json (hunk 2 above)
Comment (sstone1) [reply client/package.json]: Yes, npm automatically adds the `.bin` directory into the PATH when it runs any of the scripts.

Comment (sstone1) [reply client/package.json]: Also, the build works, so of course it does ;-)

**Pull request merged**